clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -18,36 +20,15 @@ import ViewYourDataMapPage from "./ViewYourDataMapPage";
 
 const ConfigWizardWalkthrough = () => {
   const router = useRouter();
-  const [step, setStep] = useState<number>(1);
-  const [reviewStep, setReviewStep] = useState<number>(1);
-  const [systemFidesKey, setSystemFidesKey] = useState("");
+
+  const step = useAppSelector((state) => state.configWizard.step);
+  const reviewStep = useAppSelector((state) => state.configWizard.reviewStep);
```

Comment:
Good catch.

Reply:
My only hesitation with this pattern is it requires a circular import: the slice imports `AppState` from `store.ts` which also imports from the slice file. But I think it doesn't cause any problems because it's only a _type_ import in one direction.

Reply:
oh yeah, I learned from Elliot a few weeks ago that you can just import by type in typescript https://github.com/ethyca/fidesops/pull/577#discussion_r892531819